app/src/main/graphql/checkout.graphql

```diff
@@ -78,8 +78,8 @@ mutation CompleteOnSessionCheckout($checkoutId: ID!, $paymentIntentClientSecret:
   }
 }
 
-mutation completeOrder($projectId: ID! $stripePaymentMethodId: String, $paymentSourceId: String, $paymentSourceReusable: Boolean, $paymentMethodTypes: [String!]) {
-  completeOrder(input:{ projectId: $projectId, stripePaymentMethodId: $stripePaymentMethodId, paymentSourceId: $paymentSourceId, paymentSourceReusable: $paymentSourceReusable, paymentMethodTypes: $paymentMethodTypes }) {
+mutation completeOrder($projectId: ID!, $stripePaymentMethodId: String, $paymentSourceId: String, $paymentSourceReusable: Boolean, $paymentMethodTypes: [String!]) {
```

Comment:
im a bit nervous about this change. this change was required after i downloaded the new graphql schema.